webapi/src/test/scala/org/knora/webapi/http/middleware/AuthenticationMiddlewareZSpec.scala

```diff
@@ -0,0 +1,82 @@
+package org.knora.webapi.http.middleware
+
+import zhttp.http.Http
+import zhttp.http.Request
+import zhttp.http.Response
+import zio._
+import zio.test._
+
+import org.knora.webapi.messages.admin.responder.groupsmessages.GroupADM
+import org.knora.webapi.messages.admin.responder.permissionsmessages.PermissionsDataADM
+import org.knora.webapi.messages.admin.responder.projectsmessages.ProjectADM
+import org.knora.webapi.messages.admin.responder.usersmessages.UserADM
+import org.knora.webapi.messages.util.KnoraSystemInstances
+import org.knora.webapi.routing.admin.AuthenticatorService
+
+object AuthenticationMiddlewareZSpec extends ZIOSpecDefault {
```

Comment:
```suggestion
object AuthenticationMiddlewareSpec extends ZIOSpecDefault {
```

It is a `Spec` for the `AuthenticationMiddleware`, in test source set we do support tests other than `zio-test` anyway.

Reply:
you're right, changed it. There are some more to rename eventually, all the old ones are still called `...ZSpec`